[2025-06-06T03:00:27.361+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: duration_prediction_training.train_model manual__2025-06-06T03:00:25.742029+00:00 [queued]>
[2025-06-06T03:00:27.370+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: duration_prediction_training.train_model manual__2025-06-06T03:00:25.742029+00:00 [queued]>
[2025-06-06T03:00:27.370+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-06-06T03:00:27.381+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): train_model> on 2025-06-06 03:00:25.742029+00:00
[2025-06-06T03:00:27.387+0000] {standard_task_runner.py:60} INFO - Started process 258 to run task
[2025-06-06T03:00:27.389+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'duration_prediction_training', 'train_model', 'manual__2025-06-06T03:00:25.742029+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/xgboost_duration-prediction_dag.py', '--cfg-path', '/tmp/tmpg0ehff9c']
[2025-06-06T03:00:27.391+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask train_model
[2025-06-06T03:00:27.403+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-06-06T03:00:27.432+0000] {task_command.py:423} INFO - Running <TaskInstance: duration_prediction_training.train_model manual__2025-06-06T03:00:25.742029+00:00 [running]> on host 1c35f4870e19
[2025-06-06T03:00:27.502+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='duration_prediction_training' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2025-06-06T03:00:25.742029+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-06T03:00:25.742029+00:00'
[2025-06-06T03:00:27.521+0000] {logging_mixin.py:188} WARNING - 2025/06/06 03:00:27 INFO mlflow.tracking.fluent: Experiment with name 'nyc-taxi-experiment-1' does not exist. Creating a new experiment.
[2025-06-06T03:01:13.150+0000] {logging_mixin.py:188} INFO - Index(['tpep_pickup_datetime', 'tpep_dropoff_datetime', 'PULocationID',
       'DOLocationID', 'trip_distance', 'duration', 'PU_DO'],
      dtype='object')
[2025-06-06T03:01:13.160+0000] {logging_mixin.py:188} INFO -            PU_DO  trip_distance
2904679  236_236           0.30
3144343  161_161           0.56
1278494  151_239           1.07
2578212    4_211           1.10
3345592  236_238           1.50
[2025-06-06T03:01:13.792+0000] {logging_mixin.py:188} INFO - Index(['tpep_pickup_datetime', 'tpep_dropoff_datetime', 'PULocationID',
       'DOLocationID', 'trip_distance', 'duration', 'PU_DO'],
      dtype='object')
[2025-06-06T03:01:13.798+0000] {logging_mixin.py:188} INFO -            PU_DO  trip_distance
2988209  231_249           0.59
567285    90_113           1.19
2634088   233_13           6.43
1239027   140_43           1.22
3127166  186_114           1.65
[2025-06-06T03:01:14.439+0000] {logging_mixin.py:188} WARNING - 2025/06/06 03:01:14 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2025-06-06T03:01:14.511+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/xgboost/core.py:158 UserWarning: [03:01:14] WARNING: /workspace/src/objective/regression_obj.cu:227: reg:linear is now deprecated in favor of reg:squarederror.
[2025-06-06T03:01:15.566+0000] {logging_mixin.py:188} INFO - [0]	valid-rmse:10.09408
[2025-06-06T03:01:16.476+0000] {logging_mixin.py:188} INFO - [1]	valid-rmse:9.44140
[2025-06-06T03:01:16.612+0000] {logging_mixin.py:188} INFO - MLflow Tracking URI: http://mlflow:5000
[2025-06-06T03:01:16.623+0000] {logging_mixin.py:188} INFO - MLflow Artifact URI (before logging artifact): /opt/***/mlruns/1/d7248836cff64a49b09412016fb7cbc1/artifacts
[2025-06-06T03:01:16.638+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/xgboost/core.py:158 UserWarning: [03:01:16] WARNING: /workspace/src/c_api/c_api.cc:1374: Saving model in the UBJSON format as default.  You can use file extension: `json`, `ubj` or `deprecated` to choose between formats.
[2025-06-06T03:01:18.031+0000] {logging_mixin.py:188} WARNING - 2025/06/06 03:01:18 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-06-06T03:01:18.050+0000] {logging_mixin.py:188} WARNING - 2025/06/06 03:01:18 INFO mlflow.tracking._tracking_service.client: üèÉ View run bouncy-conch-161 at: http://mlflow:5000/#/experiments/1/runs/d7248836cff64a49b09412016fb7cbc1.
[2025-06-06T03:01:18.050+0000] {logging_mixin.py:188} WARNING - 2025/06/06 03:01:18 INFO mlflow.tracking._tracking_service.client: üß™ View experiment at: http://mlflow:5000/#/experiments/1.
[2025-06-06T03:01:18.063+0000] {logging_mixin.py:188} INFO - Model trained and logged with run_id: d7248836cff64a49b09412016fb7cbc1
[2025-06-06T03:01:18.100+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-06-06T03:01:18.108+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=duration_prediction_training, task_id=train_model, execution_date=20250606T030025, start_date=20250606T030027, end_date=20250606T030118
[2025-06-06T03:01:18.180+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-06T03:01:18.201+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
